### PR TITLE
dasharo-compatibility/326-dasharo-tools-suite.md: rename file, add template for new test cases

### PR DESCRIPTION
**Pull request by Pokisiekk**
Signed-off-by: Kamil Pokornicki <kamil.pokornicki@3mdeb.com>

### CODE REVIEW

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md [hunk 1]
```diff
@@ -0,0 +1,361 @@
...
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `1` and press `Enter`.
+1. Wait for the question: Do you want to support Dasharo development by sending
+    us logs with hardware configuration?
+1. Type in `y` and press Enter.
+
+**Expected result**
+
+The report should be generated, the whole process may take a few minutes.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:71]: At this point, we should add information, that the test case ends with success if in the summary we could read that all calls exited without errors and the report has been sent to the cloud.

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md [hunk 2]
```diff
@@ -0,0 +1,361 @@
...
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `10` and press `Enter`.
+
+**Expected result**
+
+The DUT should be turned off.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:111]:
```suggestion
The DUT should be turned off without any complications.
```

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md [hunk 3]
```diff
@@ -0,0 +1,361 @@
...
+1. Plug the USB stick with DTS into the USB slot on the DUT.
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type in `11` and press `Enter`.
+
+**Expected result**
+
+The DUT should be restarted.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:140]:
```suggestion
The DUT should be rebooted without any complications.
```

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md [hunk 4]
```diff
@@ -0,0 +1,361 @@
...
+1. Select the USB stick with DTS using the arrow keys and press `Enter`.
+1. Wait for `Enter an option:`.
+1. Type 9 and click `Enter` to launch Shell.
+1. Run the following command to obtain `coreboot.rom` binary:
+
+    ```bash
+    wget https://3mdeb.com/open-source-firmware/{PATH} -O /tmp/coreboot.rom
+    ```
+
+    > This is not only way to obtain binary. For example you can use `scp`.
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:210]:
```suggestion
    > The above-described command is not the only way to obtain binary. For example, `scp` command might be used, too.
```

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+
+1. Run the following command to flash the firmware:
+
+    ```bash
+    flashrom -p internal -w /tmp/coreboot.rom
```
Comment (sulewskiprzemyslaw) [inline docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:215]: Are You sure about this command? We are going to flash all of the flash chip?

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md (hunk 5 above)
Comment (Pokisiekk) [reply docs/unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md:220]: I added information on where to find the exact command for each platform.

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**